dashboard/dashboard-front/src/components/jobs/JobsListView.vue

```diff
@@ -150,7 +152,13 @@ watch(autoUpdateEnabled, () => {
 var autoReloadSocket: Socket | null = null
 
 function setupEventStreamClient() {
+    if (!eventStreamFeatureEnabled) {
+        console.log(`Event stream feature is disabled`)
```

Comment:
It will be re-enabled as part of issue https://github.com/TheRacetrack/racetrack/issues/317